callcommands/sv_callcommands.lua

```diff
@@ -142,12 +142,22 @@
             RegisterServerEvent('SonoranCAD::callcommands:SendCallApi')
             AddEventHandler('SonoranCAD::callcommands:SendCallApi',
                 function(emergency, caller, location, description, source, silenceAlert, useCallLocation, t)
+					-- Give an unchanged varible to check if call came in as vec3
+					local vec3Call = false
+					local locationVec3 = nil
+					if type(location) == 'vector3' then 
+						vec3Call = true 
+						locationVec3 = location
+					end
+					
                     local postal = ""
                     if location == '' then
                         location = LocationCache[source] ~= nil and LocationCache[source].location or 'Unknown'
-                    elseif type(location) == 'vector3' then
-                        postal = getPostalFromVector3(location)
+                    elseif vec3Call then
+                        postal = getPostalFromVector3(location) 
+						location = "[" .. postal .."]" .. " Street Unknown" -- Set the location to something other than a print of an xyz
```

Comment:
Just noticed this. LocationCache may be blank if it's a vec3 call from a script, but postal will _always_ be blank here because it's inited as "" and defined further down the code.

Actually, this was always an issue with the initial code as the actual code is incorrect (oops). If you could, move the functions further down where it checks for the plugin load since the function comes from the postal plugin.

Reply:
I'm honestly not sure what you mean, please feel free to modify the code as you see fit